VideoApi/VideoApi.UnitTests/Services/VirtualRoom/GetOrCreateAnInterpreterVirtualRoomTests.cs

```diff
@@ -108,7 +108,7 @@ public async Task should_create_vmr_with_kinly_if_room_is_not_available()
             
             _mocker.Mock<ICommandHandler>().Setup(x =>
                 x.Handle(It.IsAny<UpdateParticipantRoomConnectionDetailsCommand>())).Callback(() =>
-                expectedRoom.UpdateConnectionDetails(newVmrRoom.Room_label, "ingesturl", newVmrRoom.Uris.Pexip_node,
+                expectedRoom.UpdateConnectionDetails(newVmrRoom.Room_label, "ingesturl_interpreter_2", newVmrRoom.Uris.Pexip_node,
```

Comment:
Instead of hard coding the 2 use expectedRoomId.

Reply:
Same for ingesturl, I don't exactly see where this is set but it comes from _conference.IngestUrl right?

Reply:
Is this not the same as expectedIngestUrl? Just it would be better to use the variable if so.